modules/op-resolver-git.adoc

```diff
@@ -0,0 +1,209 @@
+// This module is included in the following assembly:
+//
+// // *openshift_pipelines/remote-pipelines-tasks-resolvers.adoc
+
+:_content-type: PROCEDURE
+[id="resolver-git_{context}"]
+= Specifying a remote pipeline or task from a Git repository
+
+Using the Git resolver, you can specify a remote pipeline or task from a Git repostory. The repository must contain a YAML file that defines the pipeine or task.
+
+The Git resolver can access a repository either by cloning it anonymously or else by using the authenticated SCM API.
+
+For the authenticated API, you can use Git repoisitory providers that are supported by the link:https://github.com/jenkins-x/go-scm/tree/main/scm/driver[`go-scm` library].
+Not all `go-scm` implementations have been tested with the Git resolver, but the following providers are known to work:
```

Comment:
We may want to highlight a shortcoming today which is, it can only be configured for the entire cluster, making it a bit "brittle" if you want to have different team and different private repository to pull from (because you would need to create a user that has access to all).

cc @chmouel

Reply:
I believe I did this one.